.travis.yml

```diff
@@ -23,9 +23,9 @@ jobs:
     - env: GROUP=2
       python: 2.7
     - env: GROUP=1
-      python: 3.6
+      python: 3.7
```

Comment:
Add
```yaml
dist: xenial
sudo: required
```
and it will work right now

Reply:
Yep. Indeed it does. Thanks for the tip.

I'm happy to wait a bit to see if there's any activity/stability on this front -- for non-sudo/non-xenial systems -- Travis CI literally added this 10 hours back. I'd prefer to not have to change the configuration multiple times in the span of a few days. :)